build.gradle

```diff
@@ -21,9 +21,7 @@ buildscript {
     dependencies {
         classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.2'
         classpath 'org.github.ngbinh.scalastyle:gradle-scalastyle-plugin_2.10:0.8.2'
-        classpath 'com.bmuschko:gradle-nexus-plugin:2.3.1'
```

Comment:
we used this just for creating additional artefacts for nexus upload.

Currently, we create the artefacts as well in the same way but for all the repositories -> i.e -> releasing to local maven or nexus will be always consistent